libraries/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -281,7 +335,7 @@ static IotMqttError_t _deserializeIncomingPacket( _mqttConnection_t * pMqttConne
     _mqttOperation_t * pOperation = NULL;
 
     /* Deserializer function. */
-    IotMqttError_t ( * deserialize )( _mqttPacket_t * ) = NULL;
+    _deserializer_t deserialize = NULL;
```

Comment:
Is this not used? On Travis, the compiler gives unused variable warnings.

Reply:
It was not initialized, surprised not to see a compiler warning for this when building.  Removed it.